sweep/sweeper.go

```diff
@@ -1038,6 +1062,12 @@ func (s *UtxoSweeper) handlePendingSweepsReq(
 
 	resps := make(map[wire.OutPoint]*PendingInputResponse, len(s.inputs))
 	for _, inp := range s.inputs {
+		// Skip immature inputs for compatibility.
+		mature, _ := inp.isMature(uint32(s.currentHeight))
+		if !mature {
+			continue
```

Comment:
👍 
should it not also do all the other checks done by `updateSweeperInputs`?

Reply:
yeah eventually i wanna create a more verbose RPC for returning the pending sweeps - atm the goal is to stay compatible so the itests (hence the existing users) are not affects.